docs/client.rst

```diff
@@ -52,7 +52,7 @@ For incorporating the changes to your project, is necessary to run the ``cmake``
     Enables or disables the functions related to write topics.
 
 ``PROFILE_DISCOVERY=<bool>``
-    Enables or disables the functions the discovery feature.
+    Enables or disables the functions the discovery feature (currently, only for Linux).
```

Comment:
```suggestion
    Enables or disables the functions of the discovery feature (Currently, only for Linux).
```

Reply:
Why the Uppercase in `Currently`?

Reply:
You are right, that should be uppercase.
I have updated the suggestion.